Add some refs. and comments. (#38)
escience-academy/Lesson-FAIR-Data-Climate: _episodes/introduction.md

has subsequently been endorsed by working groups, funding bodies and
institutions.

> The FAIR principles have a strong focus on "machine-actionability".
> This means that the data should be primarily readable by computers (and not only by humans), which is particularly relevant for working and discovering new data.

Reviewer: ```suggestion
The FAIR principles have a strong focus on "machine-actionability".
This means that the data should be easily readable by computers (and not only by humans). This is particularly relevant for working with and discovering new data.
```

![FAIR principles illustration by Scriberia](../fig/FAIRPrinciples.jpg) *This
image was created by Scriberia for The Turing Way community and is used under a
CC-BY licence. Source: <https://doi.org/10.5281/zenodo.3695300>*

> > ## FAIR is not (necessarily) Open

Reviewer: No need to add the necessarily here imo. 
```suggestion
> ## FAIR is not open
```

>

> > Some data cannot be openly shared, for example, because of privacy considerations.
> > For such situations, the EU H2020 Program Guidelines on FAIR Data apply:

Reviewer: ```suggestion
> Some data cannot be shared openly, for example, because of privacy considerations.
> For such situations, the EU H2020 Program Guidelines on FAIR Data say that the data should be *"As open as possible, as closed as necessary."*
```

>

> > data should be *"As open as possible, as closed as necessary."*

Reviewer: ```suggestion
```

> ## What's in it for you?
>

> > Many funders nowadays require from researchers data management plans which
> > adhere to the FAIR principles. A great step forward, but if 

Reviewer: ```suggestion
> Many funders nowadays require from researchers to write data management plans which
> adhere to the FAIR principles. A great step forward, but if 
```

> we are going to spend the time and effort, we prefer to find some intrinsic 
> motivation.
>
> Have a look at the original [Nature paper](https://doi.org/10.1038/sdata.2016.18)
> on FAIR Data, the [GO-FAIR initiative](https://www.go-fair.org/fair-principles/),
> and the Danish tutorial [howtofair.dk](https://www.howtofair.dk/why-fair/).
> What are the main reasons for *you* to (not) embrace the FAIR principles?
{: .discussion}

> ## How FAIR are my data?
> _AK: The title was a bit misleading; there isn't much about FAIRification process itself._

Reviewer: ```suggestion
## How FAIR are my data?
```

Reviewer 2: @stefsmeets this suggestion is made to the wrong line I think. Cannot commit it. Can you change it?
